README.md

```diff
@@ -20,40 +27,17 @@ Golang Library packages for [IndyKite Platform](https://indykite.com) Client SDK
 import "github.com/indykite/jarvis-sdk-go"
 ```
 
-In order to access to the platform you must obtain an API key first.
+In order to access to the platform you must first obtain AppAgent credentials.
 This key can be obtained either from the [Admin Console](https://console.indykite.id)
 or request one from your point of contact at IndyKite.
 
 ## Requirements
 
-Go 1.19
-
-## Used terminology
-
-| Definition               | Description                                                                                      |
-|--------------------------|--------------------------------------------------------------------------------------------------|
-| Digital Twin             | A digital twin is the digital identity of a physical entity on/in a software/identity system     |
-| Application Space ID     | ID of the application where the digital twin belongs to                                          |
-| Application Agent ID     | ID of the agent which makes the application available for the different calls                    |
-| Tenant ID                | ID of the tenant where the digital twin belongs to. The tenant is belong to an application space |
-| Private Key and Settings | The secret which required to reach the system. Indykite provides the necessary secrets           |
-| Property                 | The digital twin's property (eg.: email, name)                                                   |
-| JWT                      | JSON Web Tokens                                                                                  |
-| Introspect               | A process used to validate the token and to retrieve properties assigned to the token            |
-| Patch property           | Add, change or delete a property of a digital twin                                               |
+- Go version 1.19 or later
+- An AppSpace, App, AppAgent, and AppAgent credentials
+- Service account credentials
```

Comment:
I don't think a Service Account is strictly necessary to set up before using

Reply:
Well, AppSpace + App + AppAgent + App Agent credentials might not be needed too

- You need AppAgent credentials if using Identity, Ingest, Knowledge API etc. Basically everything except Config API
- You need Service Account when using Config API

Reply:
I took what was in the current Go SDK documentation in docs.indykite (https://docs.indykite.com/docs/quick-starts/sdks/go-sdk)

We can just use "Go version 1.19 or later" and take the other items out, or at least specify what credentials are needed for what API. Here and in the doc set (not just for Go SDK, but for the others as well).

Reply:
I'm OK with having some small description here. I was just saying that you don't need both. You might need one, second or both. Depends on the usage